### PR TITLE
feat: create aggregated GatewayApiClient for SDK

### PR DESCRIPTION
**Pull request by dawidsowardx**

### CODE REVIEW

File: sdk/typescript/README.md [hunk 1]
```diff
@@ -1,25 +1,95 @@
...
+
+Calling static `intialize` method from `GatewayApiClient` class will instantiate all low-level APIs together with their high-level equivalents and return them as a single object.
 
 ```typescript
-import "./node-fetch-polyfill" // Polyfill for fetch required if running in node-js
-import { StateApi, StatusApi, TransactionApi, StreamApi } from "@radixdlt/babylon-gateway-api-sdk";
+import { GatewayApiClient } from '@radixdlt/babylon-gateway-api-sdk'
+
+const gatewayApi = GatewayApiClient.initialize({
+  basePath: 'https://kisharnet-gateway.radixdlt.com',
```
Comment (dhedey) [inline sdk/typescript/README.md:24]: We should prefer `https://rcnet.radixdlt.com` :)

File: sdk/typescript/README.md [hunk 2]
```diff
@@ -1,25 +1,95 @@
...
+- `getEntityDetailsVaultAggregated(entities: string | string[])` - detailed information about entities
+
+### Status
+
+- `getCurrent()` - Gateway API version and current ledger state
+- `getNetworkConfiguration()` - network identifier, network name and well-known network addresses
+
+### Transaction
+
+- `getStatus(txID: string)` - transaction status for given transaction id.
```
Comment (dhedey) [inline sdk/typescript/README.md:79]: We should probably say `transaction id (the intent hash)` - just to align with the language that the toolkit is using.

**Review by dhedey — APPROVED** [review]
A few doc tweaks suggested, but otherwise really nice thanks! 👍

File: sdk/typescript/README.md [hunk 3]
```diff
@@ -1,25 +1,95 @@
...
+  basePath: 'https://kisharnet-gateway.radixdlt.com',
+})
+```
+
+## Low Level APIs
+
+Low level APIs are generated automatically based on OpenAPI spec. You can get a good sense of available methods by looking at [Swagger](https://rcnet.radixdlt.com/swagger/index.html). There are two ways of using each sub api
 
-const transactionApi = new TransactionApi();
+### Using generated APIs through `innerClient`
```
[the remark this answers is not in the document]
Comment (dawidsowardx) [reply sdk/typescript/README.md:37]: marked `lowLevel` as private

**Pull request merged**